x-pack/plugins/security_solution/public/detections/components/rules/related_integrations/use_installed_integrations.tsx

```diff
@@ -18,23 +18,25 @@ export interface UseInstalledIntegrationsArgs {
 export const useInstalledIntegrations = ({ packages }: UseInstalledIntegrationsArgs) => {
   const { addError } = useAppToasts();
 
-  return useQuery<GetInstalledIntegrationsResponse>(
+  return useQuery<InstalledIntegrationArray>(
     [
       'installedIntegrations',
       {
         packages,
       },
     ],
     async ({ signal }) => {
-      return fetchInstalledIntegrations({
+      const integrations = await fetchInstalledIntegrations({
         packages,
         signal,
       });
+      return integrations.installed_integrations ?? [];
     },
     {
       keepPreviousData: true,
       onError: (e) => {
-        addError(e, { title: i18n.INSTALLED_INTEGRATIONS_FETCH_FAILURE });
+        // TODO: Suppress if always failing in cases where fleet isn't setup, etc
+        addError(e, { title: i18n.INTEGRATIONS_FETCH_FAILURE });
```

Comment:
I'm thinking we should just suppress this error toast for now until we have a chance to test all the RBAC scenarios on when/where it can fail that we've outlined in the test plan. Would rather fail silently here than spam certain deployments that have limited privileges, access to fleet, etc...

Reply:
@spong totally agree!